robot-command/src/main/java/org/obolibrary/robot/FilterCommand.java

```diff
@@ -212,12 +216,40 @@ public CommandState execute(CommandState state, String[] args) throws Exception
           outputOntology, RelatedObjectsHelper.getAnnotationAxioms(inputOntology, relatedObjects));
     }
 
+    // drop specified axiom annotations
+    dropAxiomAnnotations(line, ioHelper, outputOntology);
+
     // Save the changed ontology and return the state
     CommandLineHelper.maybeSaveOutput(line, outputOntology);
     state.setOntology(outputOntology);
     return state;
   }
 
+  /**
+   * Given a command line and an IOHelper, drops axiom annotations from the given ontology which
+   * uses annotation properties specified by the drop_axiom_annotation argument.
+   *
+   * @param line CommandLine to get options from
+   * @param ioHelper IOHelper to get IRIs
+   * @param ontology OWLOntology to drop axiom annotations from
+   */
+  private void dropAxiomAnnotations(CommandLine line, IOHelper ioHelper, OWLOntology ontology) {
+    OWLOntologyManager manager = ontology.getOWLOntologyManager();
+    List<String> annotationsToDrop =
+        CommandLineHelper.getOptionalValues(line, "drop_axiom_annotation");
```

Comment:
keb.

Reply:
fixed and method moved to `RelatedObjectsHelper.java`